bescort.lic

```diff
@@ -1152,7 +1202,7 @@ class Bescort
 
   def segoltha(mode)
     EquipmentManager.new.empty_hands
-    have_changed_gear = EquipmentManager.new.wear_equipment_set?('swimming')
+    have_changed_gear = EquipmentManager.new.wear_equipment_set?('swimming') unless @flying_mount
```

Comment:
This makes the assignment conditional, which can leave `have_changed_gear` as nil. While `nil` is falsey, I think it would be clearer to make this assignment always happen by including `@flying_mount` in the boolean calculation.

Something like `have_changed_gear = !@flying_mount && EquipmentManager.new.wear_equipment_set?('swimming')`.

What do you think?

Reply:
I ended up using your favorite ternary operator to get around both issues.
`have_changed_gear = flying_mount ? false : EquipmentManager.new.wear_equipment_set?('swimming')`